applications/IgaApplication/custom_utilities/brep_topology/brep_edge.cpp

```diff
@@ -0,0 +1,68 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:     BSD License
+//           Kratos default license: kratos/IGAStructuralMechanicsApplication/license.txt
+//
+//  Main authors:    Tobias Teschemacher
+//                   Michael Breitenberger
+//                   Thomas Oberbichler
```

Comment:
You can drop my name since I haven't done nothing inside this file and I already have a lot of other stuff to maintain :P

Reply:
Haha you can not escape from this.